### PR TITLE
Update the design of the main Suggested Edits screen.

### PR DESCRIPTION
**Pull request by dbrant**
https://phabricator.wikimedia.org/T250202

### CODE REVIEW

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -352,6 +352,13 @@
         <item name="android:textColor">@color/red50</item>
     </style>
 
+    <style name="ToolbarTextAppearance">
+        <item name="android:textColor">?attr/material_theme_primary_color</item>
+        <item name="android:textSize">24sp</item>
+        <item name="android:textStyle">bold</item>
+    </style>
+
+
```
Comment (cooltey) [inline app/src/main/res/values/styles.xml:361]: empty line.

File: app/src/main/res/layout-sw600dp/view_suggested_edits_task_item.xml [hunk 2]
```diff
@@ -97,25 +89,29 @@
...
         android:layout_height="48dp"
-        android:padding="4dp"
+        android:layout_marginStart="16dp"
+        android:background="@drawable/button_shape_wide_corners"
+        android:backgroundTint="?android:attr/colorBackground"
         android:clickable="true"
         android:focusable="true"
-        android:layout_marginStart="16dp"
-        app:layout_constraintTop_toTopOf="@id/addContainer"
+        android:foreground="?attr/selectableItemBackground"
```
Comment (cooltey) [inline app/src/main/res/layout-sw600dp/view_suggested_edits_task_item.xml:110]: I noticed the ripple animation does not round as the button itself, should we fix that?

File: app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTaskView.kt [hunk 3]
```diff
@@ -4,23 +4,29 @@ import android.content.Context
...
+internal class SuggestedEditsTaskView constructor(context: Context, attrs: AttributeSet? = null) : CardView(context, attrs) {
 
     init {
+        val params = LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.WRAP_CONTENT)
+        params.topMargin = DimenUtil.roundedDpToPx(8f)
+        params.bottomMargin = params.topMargin
+        params.leftMargin = DimenUtil.roundedDpToPx(16f)
+        params.rightMargin = params.leftMargin
+        layoutParams = params
+        setCardBackgroundColor(ResourceUtil.getThemedColor(getContext(), R.attr.paper_color))
```
Comment (cooltey) [inline app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTaskView.kt:25]: Not sure if it is the expected design, but the cards can be barely seen in the dark/black mode.

File: app/src/main/res/values/styles.xml (hunk 1 above)
Comment (dbrant) [reply app/src/main/res/values/styles.xml]: ok!

File: app/src/main/res/layout-sw600dp/view_suggested_edits_task_item.xml (hunk 2 above)
Comment (dbrant) [reply app/src/main/res/layout-sw600dp/view_suggested_edits_task_item.xml]: Fixed!

File: app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTaskView.kt (hunk 3 above)
Comment (dbrant) [reply app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTaskView.kt:25]: Let's let Robin comment on it during his review.

**Pull request merged**